campuses-with-active-researchers/campuses-with-active-researchers.py

```diff
@@ -138,13 +138,16 @@ def main(argv):
     if starttime > endtime:
         parser.error("Start date can't be after end date")
 
+    runtime = datetime.datetime.now()
     active_organizations = get_organizations_with_active_researchers(starttime, endtime)
     ccstar_facilities = get_ccstar_facilities()
 
     # TODO: Project Organizations do not necessarily match Topology Facilities.
     # This may result in errors in the CC* column (false negatives being more likely).
     # Not sure where the place to fix it is -- or if the comparison is even meaningful.
     if args.csv:
+        nowts = runtime.strftime("%F at %H:%M")
+        dateinfo = "(Generated on {} for {} through {})".format(nowts, args.startdate, args.enddate)
```

Comment:
Where does `dateinfo` get printed/used?

Reply:
Good catch!  This got lost in some intermediate development steps...

Updated to (somewhat lamely) include the dateinfo in the csv header.  (For lack of a better place for it.)

Reply:
@matyasselmeci thoughts since it's your script?

@edquist if subsequent rows only have two fields, will that cause issues if there are three headers for any CSV import tools?

Reply:
>if subsequent rows only have two fields, will that cause issues if there are three headers for any CSV import tools?

That'd be super lame if they did!

Idk, the ones i use don't complain about it.

Reply:
PS
> @matyasselmeci thoughts since it's your script?

how considerate  :)

Reply:
Maybe put the dateinfo (and any other explanatory text) on row 1 and move everything else down a row?

Reply:
If you feel strongly that that's the way to go, we can do that.  But in general i think spreadsheet programs interpret the first row as the header row with the column names, and spreadsheety functions like "sorting" sort the remaining rows based on the column names in the first row.

So, i am not wild about putting the column headers on the second row.  It might look pretty enough if all you do is look at it, but if you do anything spreadsheety it seems this will just be annoying.  Though i suppose we could also ask Janet (who will be looking at it) or @bbockelm (who requested this feature).

(You might say adding an extra non-column header-row field is awkward for a spreadsheet also - and i would agree - but at least having a column with empty cells does not mess up the column name detection and other normal spreadsheet operations like sorting or counting.)

Anyway i would feel amiss not expressing these thoughts of mine, but having gotten them out, i'll be fine with however you want to handle it.  (Or if you think i'm making a mess of your script feel free to make a PR against mine  :)

Reply:
I figured skipping a line would be pretty easy for any importer to deal with but I'm not the one running the script so I have no objections either way.

Reply:
>I figured skipping a line would be pretty easy for any importer to deal with

yeah for instance i use libreoffice and when importing it does have an option for which row to start with (default 1).  So you can start with row 2 and get just the regular column headers at the top, but then you don't see the date/time info at all in the loaded spreadsheet.

>I'm not the one running the script

I'm not the one that's gonna be looking at the spreadsheet either - i'll ask janet if she has a preference